### PR TITLE
Switch to using BroadcastEvent in the feed itself

### PR DESCRIPTION
**Pull request by beccahughes**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/media-feeds/pull/21.html" title="Last updated on Apr 17, 2020, 4:42 PM UTC (a67d422)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/media-feeds/21/9399530...a67d422.html" title="Last updated on Apr 17, 2020, 4:42 PM UTC (a67d422)">Diff</a>

### CODE REVIEW

**Comment by beccahughes** [comment]
We should add the `isLiveBroadcast` boolean to our live details mojo since not all broadcast events are actually live (e.g. prerecorded tv)

File: explainer.md [hunk 1]
```diff
@@ -325,45 +320,46 @@ If a site wishes to recommend a TV series to a user they can use the [TVSeries](
 
 ### A site recommends some live video content to a user
 
-If a site wishes to recommend some live video content to a user they can add a [BroadcastEvent](https://schema.org/BroadcastEvent) in the publication property (this is also on the TVEpisode and Movie media item types as well). The event should have the `isLiveBroadcast` property set to true and the `startDate` property set to the time the broadcast started. The `endDate` should be included too if known.
+If a site wishes to recommend a generic video to a user they can add a [BroadcastEvent](https://schema.org/BroadcastEvent) to the feed. This is a special type that describes a broadcast schedule or a live broadcast and embeds the media item being broadcast in the `workPerformed` property.
```
Comment (sgbowen) [inline explainer.md:323]: I don't understand what "generic video" means here. It's not actually generic compared to the other types of content, right? It's just a specific type. I like the description in the second sentence, but do you think you could reword this first part somehow?

File: index.html [hunk 2]
```diff
@@ -492,6 +463,113 @@ <h3>Media Feed Item</h2>
...
+            action</a> unless the object has an <a>embedded watch action</a>.
+          </li>
+          <li>
+            The <a data-cite="schema-org/startDate#">startDate</a> property MUST
+            contain a string that is valid date or date time in
+            <a data-cite="ISO8601#">ISO8601 format</a>.
+          </li>
+          <li>
+            The <a data-cite="schema-org/endDate#">endDate</a> property MAY
+            be present. It it is, it MUST contain a string that is valid
```
Comment (sgbowen) [inline index.html:510]: If it is*

File: explainer.md [hunk 3]
```diff
@@ -144,7 +143,6 @@ If the media item url will not resume playback at the correct time, then the `ta
     "url": "https://example.org/testuser"
   },
   "datePublished": "2020-01-27",
-  "description": "This is a video that is about a bunny."
```
Comment (sgbowen) [inline explainer.md:147]: Why remove these descriptions?

Comment (beccahughes) [reply explainer.md:170]: They are not needed since they are not part of the spec

File: explainer.md (hunk 1 above)
Comment (sgbowen) [reply explainer.md]: Thanks for the rewording. Looks better to me.

**Pull request merged**